3rd/cjson/include/cjson.hpp

```diff
@@ -0,0 +1,91 @@
+#include "cjson/cJSON.h"
+#include "pocketpy/pocketpy.h"
+
+
+namespace pkpy{
+
+static Dict convert_cjson_to_dict(const cJSON * const item, VM* vm)
+{
+
+    Dict output(vm);
+    List list;
+
+    if ((item == NULL))
+    {
+        return NULL;
+    }
+
+    if (((item->type) & 0xFF) == cJSON_Object){
+            cJSON *child = item->child;
+            while(child != NULL){
+                const cJSON *child_value = cJSON_GetObjectItemCaseSensitive(item, child->string);
+           
+                if (cJSON_IsString(child_value))
+                {
+                    output.set(VAR(Str(child->string)), VAR(Str(child_value->valuestring)));
+                }
+                else if (cJSON_IsNumber(child_value)){
+                    output.set(VAR(Str(child->string)), VAR(child_value->valueint));
+                }
+                else if (cJSON_IsBool(child_value)){
+                    output.set(VAR(Str(child->string)), VAR(child_value->valueint)); //Todo: Covert to python boolean
```

Comment:
@blueloveTH I need your help.
I need to convert c `int` (value 1) to PythonObject (boolean value `True`). Is there any function to do that?
I also need Python `None` value as PythonObject.

Reply:
I figured out how to convert c int to python boolean. However, I still need help to convert c `NULL` to python `None`

Reply:
`None`, `False` and `True` are singleton objects! You can directly get them in `VM* vm`.

```cpp
VM* vm;
PyObject* true_object = vm->True;
PyObject* false_object = vm->False;
PyObject* none_object = vm->None;
```

For true and false, you can also use `VAR` macro.
```cpp
PyObject* true_object_2 = VAR(true);
PyObject* false_object_2 = VAR(false);
```

To convert a int into bool object,
```cpp
PyObject* convert(int i){
    return i!=0 ? vm->True : vm->False;
}
```

Reply:
Thank you so much!